### PR TITLE
[quant][pt2e] Update save/load model for pt2 export quant tutorial

### PR DESCRIPTION
**Pull request by jerryzh168**
Summary:
att

Test Plan:
visual inspection of generated pages

Reviewers:

Subscribers:

Tasks:

Tags:

Fixes #ISSUE_NUMBER

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Comment by jerryzh168** [comment]
generated doc looks good I think: https://docs-preview.pytorch.org/pytorch/tutorials/2539/prototype/pt2e_quant_ptq_static.html#save-and-load-quantized-model

File: prototype_source/pt2e_quant_ptq_static.rst [hunk 1]
```diff
@@ -569,9 +581,10 @@ Lowering and Performance Evaluation
 
 The model produced at this point is not the final model that runs on the device,
 it is a reference quantized model that captures the intended quantized computation
-from the user, expressed as ATen operators, to get a model that runs on real
-devices, we'll need to lower the model. For example for the models that run on
-edge devices, we can lower to executorch.
+from the user, expressed as ATen operators and some additional quantize/dequantize operators,
+to get a model that runs on real devices, we'll need to lower the model.
+For example for the models that run on edge devices, we can lower with delegation and executorch runtime
```
Comment (svekars) [inline prototype_source/pt2e_quant_ptq_static.rst:586]:
```suggestion
For example, for the models that run on edge devices, we can lower with delegation and ExecuTorch runtime
```

File: prototype_source/pt2e_quant_ptq_static.rst [hunk 2]
```diff
@@ -569,9 +581,10 @@ Lowering and Performance Evaluation
...
+operators..
```
Comment (svekars) [inline prototype_source/pt2e_quant_ptq_static.rst:587]:
```suggestion
operators.
```

File: prototype_source/pt2e_quant_ptq_static.rst [hunk 3]
```diff
@@ -519,46 +523,54 @@ Save and Load Quantized Model
...
-    loaded_quantized_model = convert_pt2e(prepared_model)
 
-    # load the state_dict from saved file to intialized model
-    loaded_quantized_model.load_state_dict(torch.load(pt2e_quantized_model_file_path))
+    # 2. Load the saved ExportedProgram
+    loaded_quantized_ep = torch.export.load(pt2e_quantized_model_file_path)
+    loaded_quantized_model = loaded_quantized_ep.module()
 
-    # Sanity check with sample data
+    # 3. Check results for example inputs and checke evaluation accuracy again
```
Comment (svekars) [inline prototype_source/pt2e_quant_ptq_static.rst:547]:
```suggestion
    # 3. Check results for example inputs and check evaluation accuracy again:
```

File: prototype_source/pt2e_quant_ptq_static.rst [hunk 4]
```diff
@@ -519,46 +523,54 @@ Save and Load Quantized Model
...
 
-.. code-block:: python
 
-    # 1. Save state_dict
-    pt2e_quantized_model_file_path = saved_model_dir + "resnet18_pt2e_quantized.pth"
-    torch.save(quantized_model.state_dict(), pt2e_quantized_model_file_path)
+.. code-block:: python
 
-    # Get a reference output
+    # 0. Store reference output for example inputs and check evaluation accuracy
```
Comment (svekars) [inline prototype_source/pt2e_quant_ptq_static.rst:529]:
```suggestion
    # 0. Store reference output, for example, inputs, and check evaluation accuracy:
```

File: prototype_source/pt2e_quant_ptq_static.rst [hunk 5]
```diff
@@ -508,6 +508,10 @@ Now we can compare the size and model accuracy with baseline model.
    target device, it's just a representation of quantized computation in ATen
    operators.
 
+.. note::
+   The weights are still in fp32 right now, we may do constant propagation for quantize op to
+   get integer weights in the future
```
Comment (svekars) [inline prototype_source/pt2e_quant_ptq_static.rst:513]:
```suggestion
   get integer weights in the future.
```

**Review by svekars — APPROVED** [review]
